src/main/java/nl/hsac/fitnesse/fixture/slim/HttpTest.java

```diff
@@ -872,6 +901,7 @@ public boolean stopTestOnException() {
 
     // Polling
     public boolean repeatUntilResponseStatusIs(final int expectedStatus) {
+        doNotThrowExceptionOnHttpRequestFailure();
```

Comment:
Changing the value of the setting seems like an unexpected side effect of repeatUntil.
Please ensure the toggle is set back to its previous setting after the 'repeat'

I actually believe a user that would want to use the repeat feature probably set the 'do not throw' setting before the first call (before the initial call that is repeated by calling one of the 'repeat' methods). So toggling here in code can probably be removed altogether.